### DISCUSSION TITLE
Next release?

### DISCUSSION THREAD

**Discussion by desruisseaux** [OSGeo/PROJ-JNI#67] [Q&A] [answered]
We have done only one [PROJ-JNI formal release](https://github.com/OSGeo/PROJ-JNI/releases/tag/1.0), which was almost 3 years ago. That release was still using the `kortforsyningen` package name. We have never done a release with the new `org.osgeo.proj` package name yet. Should we do a release this month?

If yes, what should be the version number? The next release is currently labelled as "1.1" in GitHub, but given that a change of package name is a significant change, should it be 2.0?

**Comment by kbevers** [comment]
> PROJ <PROJ@lists.osgeo.org>

That's probably a good idea, yes. Use the opportunity to introduce other breaking changes if needed.

**Comment by desruisseaux** [comment]
Changed the version number to `2.0-SNAPSHOT` and set a tentative release date to February 17 (in almost 2 weeks). Hopefully that would be enough for getting feedbacks if something broke.

* [List of closed issues](https://github.com/OSGeo/PROJ-JNI/milestone/3?closed=1)

I do not expect any breaking change other than the package renaming.

**Comment by kbevers** [comment]
> I do not expect any breaking change other than the package renaming.

Why not deal with the Java 8 issue https://github.com/OSGeo/PROJ-JNI/issues/65 while you have the chance? If you need more time to do that a new release can wait a bit longer.

**Comment by desruisseaux** [comment]
Java 8 support is a complication for maintainers, but transparent for users. A little bit like PROJ having `make` and `cmake` build systems in parallel for some time. Dropping `make` support is transparent for those who already use `cmake`.

The reason for not dropping Java 8 support right now is because we didn't asked to users. On the Apache Sedona mailing list (a project bringing geospatial services to Spark), the developers still want Java 8 for again a few years. That particular project does not use PROJ, but I do not know if there is other projects around in that situation.

We could release 2.0 as it stands today and put a note in the description saying that if nobody objects, it will be the last release supporting Java 8.

**Comment by kbevers** [comment]
> We could release 2.0 as it stands today and put a note in the description saying that if nobody objects, it will be the last release supporting Java 8.

Sounds good.

**Comment by desruisseaux** [comment, marked as answer]
PROJ-JNI 2.0 [released](https://github.com/OSGeo/PROJ-JNI/releases/2.0) with a comment saying that it may be the last release supporting Java 8. Version number on the Git repository has been set to 2.1-SNAPSHOT for next development cycle.
